dcos_test_utils/dcos_cli.py

```diff
@@ -92,6 +94,44 @@ def clear_cli_dir():
         if os.path.exists(path):
             shutil.rmtree(path)
 
+    def exec(self, cmd: str, stdin=None, check=True) -> subprocess.CompletedProcess:
+        """Execute CLI command and processes result.
+
+        This method expects that process won't block.
+
+        :param cmd: Program and arguments
+        :type cmd: str
+        :param stdin: File to use for stdin
+        :param check: Does it check for raised errors
+        :type stdin: File
+        :returns: A tuple with stdout and stderr
+        :rtype: subprocess.CompletedProcess
+        """
+
+        log.info('CMD: {!r}'.format(cmd))
+
+        try:
+            process = subprocess.run(
+                cmd,
+                stdin=stdin,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                env=self.env,
+                check=check)
+        except subprocess.CalledProcessError as e:
+            if e.stderr:
+                stderr = e.stderr.decode('utf-8')
+                log.error('STDERR: {}'.format(stderr))
```

Comment:
What about stdout?

Reply:
You would like to have a log of the STDOUT also during the error? I think it is a nice to have I will make it.

Reply:
> You would like to have a log of the STDOUT also during the error?

yes, I am afraid that some commands may print errors and/or valuable info to STDOUT as well.